config/routes.rb

```diff
@@ -1,6 +1,18 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
-  resources :line_items, :carts, :orders
+  get 'admin' => 'admin#index'
+
+  controller :sessions do
+    get 'login' => :new
+    post 'login' => :create
+    delete 'logout' => :destroy
+  end
+
+  get 'sessions/create'
+  get 'sessions/destroy'
```

Comment:
These aren't needed when declaring routes to the actions inside of the `controller :sessions` block.
```suggestion

```